src/js/letters/components/Address.jsx

```diff
@@ -153,8 +153,11 @@ const addressShape = PropTypes.shape({
 });
 
 Address.propTypes = {
-  value: addressShape.isRequired,
+  onInput: PropTypes.func.isRequired,
+  address: addressShape.isRequired,
```

Comment:
Woops! I changed these in a commit I made on the validation branch...and never pushed up because I got distracted trying to get my local environment to work.

Thanks for catching this!